Sources/jsonfunctions/JSON/Date.swift

```diff
@@ -1,7 +1,5 @@
 // Copyright (c) 2021 Patrick Amrein <amrein@ubique.ch>
-// 
-// Licensed under LGPL
```

Comment:
Is this ok?

Reply:
I checked it with open source team.. They confirmed it.
Complete original repo isn't under LGPL anymore nut under MIT.